core/src/main/kotlin/fr/sncf/osrd/utils/OffsetConversions.kt

```diff
@@ -37,8 +39,8 @@ fun trainPathZonePathOffset(
     infra: RawInfra,
     zonePaths: List<ZonePathId>,
     chunkPath: ChunkPath
-): Offset<Path> {
-    var prevChunksLength = Offset<Path>(0.meters)
+): Offset<TravelledPath> {
```

Comment:
These functions *could* have their distinct return types (zone path / chunk path), but not in this PR